src/validator/data_model.py

```
@@ -70,9 +70,72 @@ def validate_version(cls, v):
+    zone_outside_convection_algorithm: str = Field("DOE-2", alias="Zone Outside Convection Algorithm", description="Zone outside convection algorithm")
+    part_of_total_floor_area: str = Field("Yes", alias="Part of Total Floor Area", description="Part of total floor area")
     
     @field_validator('name')
     def validate_name(cls, v):
         if not v:
             raise ValueError("Name must not be empty.")
+        return v
+    @field_validator('direction_of_relative_north')
+    def validate_direction_of_relative_north(cls, v):
+        if v is not None and not (0 <= v <= 360):
+            raise ValueError("Direction of Relative North must be in [0, 360).")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)